poplar.mk

```diff
@@ -120,8 +120,8 @@ optee-os-clean: optee-os-clean-common
 l-loader: arm-tf
 	cp $(BL1_BIN) $(LLOADER_PATH)/atf
 	cp $(FIP_BIN) $(LLOADER_PATH)/atf
-	$(MAKE) -C $(LLOADER_PATH) CROSS_COMPILE="$(AARCH32_CROSS_COMPILE)"
-
+	$(MAKE) -C $(LLOADER_PATH) CROSS_COMPILE="$(AARCH32_CROSS_COMPILE)" \
+		ARM_TRUSTED_FIRMWARE="$(TF_A_PATH)"
```

Comment:
I'd rather we keep the current name: `ARM_TRUSTED_FIRMWARE`, avoids a dependency on the same change being erged in upstream l-loader.